data.sql

```diff
@@ -35,3 +35,56 @@ UPDATE animals SET owner_id = 4 WHERE name = 'Blossom';
 UPDATE animals SET owner_id = 5 WHERE name = 'Angemon';
 UPDATE animals SET owner_id = 5 WHERE name = 'Boarmon';
 
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23');
+
```

Comment:
- [ ] Insert the following data for vets:
Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
You have inserted only one data for vets please insert other as well